.github/dockerbuild.yaml

```diff
@@ -0,0 +1,23 @@
+---
+name: image-builder
+
+on:
+  push:
+    branches: ['*']
+  pull_request:
+    branches: ['*']
+
+jobs:
+  docker-build:
+    name: docker-build
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.16
+
+      - uses: actions/checkout@v3
```

Comment:
Please checkout first, so the entirety of the project can be used in the next steps